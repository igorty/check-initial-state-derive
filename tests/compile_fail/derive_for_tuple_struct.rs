use check_initial_state_derive::CheckInitialState;

fn main() {}

#[derive(CheckInitialState)]
struct Tuple (i32, Option<i32>);
