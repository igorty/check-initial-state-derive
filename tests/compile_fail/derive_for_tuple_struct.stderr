error: `CheckInitialState` procedural macro is allowed for structs with named fields only
 --> $DIR/derive_for_tuple_struct.rs:6:14
  |
6 | struct Tuple (i32, Option<i32>);
  |              ^^^^^^^^^^^^^^^^^^
